### ISSUE TITLE
Node shape not rendering when selected if ShapeId is "function"

### ISSUE THREAD

**Issue by thinkty** [uber/react-digraph#227]
**Describe the bug**
In GraphConfig.js, when the prop *id* in the symbol component is given a value "function", the node will not render properly in [renderNode](https://github.com/uber/react-digraph/blob/e154d2c5769d56ee997eec3e000979bc52994d4d/src/components/node.js#L326). 

**To Reproduce**
0. Go to [GraphConfig.js](https://github.com/uber/react-digraph/blob/e154d2c5769d56ee997eec3e000979bc52994d4d/src/examples/graph-config.js)
1. Change one of the shape's symbol's *id* prop as "function"
2. Make sure to also change *shapeId* down below in *export default* section as "#function"
3. Start the application (ex: npm start)
4. Select the node of type "function" and see error

**Expected behavior**
The node should render the svg specified in [GraphConfig.js](https://github.com/uber/react-digraph/blob/e154d2c5769d56ee997eec3e000979bc52994d4d/src/examples/graph-config.js)

**Screenshots**
Graph when nothing is selected
![normal-graph-with-nothing-selected](https://imgur.com/LfNOlPn.png)

The expected result when node of type "function" has been selected
![expected-result-when-node-selected](https://imgur.com/mz9jqIM.png)

The buggy result
![buggy-result](https://imgur.com/hyRNcYQ.png)

**Desktop (please complete the following information):**
 - OS: Ubuntu 18.04.4 LTS
 - Browser [e.g. chrome, safari] : Firefox
 - Version [e.g. 22] : 77.0.1

**Additional context**
I tested this bug with custom svg shapes and a custom renderNode method. However, other nodes render just fine and the renderNode method isn't that different to the one in [src/components/node.js](https://github.com/uber/react-digraph/blob/e154d2c5769d56ee997eec3e000979bc52994d4d/src/components/node.js#L294)

**Comment by thinkty** [comment]
As a work around, I just use "function-node" as the ID instead of "function" but I don't know why this bug occurs if the ID is "function". I'm not so familiar with React and JS but if someone wants to see my code for renderNode and the shape in GraphConfig.js, please let me know.

**Comment by thinkty** [comment]
Just tested this with the example from [here](https://github.com/uber/react-digraph/tree/master/src/examples) and the problem doesn't seem to appear.

**Comment by ajbogh** [comment]
@thinkty Can you reproduce this in the examples by running `npm run example`? I was able to render a node with the ID of "function" using the example files.

If you can recreate the issue in codesandbox.io then I'd be happy to take a look and offer more advice.

![Screenshot_2020-06-26_01-00-59](https://user-images.githubusercontent.com/161761/85834765-c93d7d80-b748-11ea-91ca-54964fcb9f5b.png)

![Screenshot_2020-06-26_01-01-12](https://user-images.githubusercontent.com/161761/85834770-ccd10480-b748-11ea-8bc8-b8da05d55158.png)

![Screenshot_2020-06-26_01-01-19](https://user-images.githubusercontent.com/161761/85834779-cf335e80-b748-11ea-801c-cbcd36ab75b7.png)

**Comment by thinkty** [comment]
I also realized that the problem doesn't appear in the example given in this repository.
I tried to run the example with and without my custom methods to render node and texts but the error did not occur.
I also tried running the example with my custom shapes but could not reproduce the error too.

I think there are some codes in my source that are not directly related to render the node that are causing this problem.

For now, I will close this until I find a way to reproduce this from the example.

Thank you
